content/src/fundings/njeda-aspire.md

```diff
@@ -44,13 +44,17 @@ sidebarCardBodyText: Aspire is a gap financing tool to support commercial, mixed
 - The project must in a net positive benefit to the State
 - Must meet specific cost thresholds, depending on where the project is located
 
-> ### Benefits
->
-> - Total award cap of $42M, but residential projects also receiving Low-Income Housing Tax Credits (LIHTC), or projects located in certain economically disadvantaged locations may receive up to $60M
-> - Tax credits equal to 45% of project costs up to $42M
->
-> ### Bonuses
->
-> - Commercial projects in a Government Restricted Municipalities (GRM) (These municipalities include: Atlantic City, Paterson and Trenton) can receive tax credits up to 50% of costs in project support
-> - Newly constructed residential projects that are also utilizing 4% LIHTCs can receive tax credits for up to 60% of project costs
-> - Projects using LIHTC or any other project in a `Qualified Incentive Track|qit-njeda` (QIT), GRM, or municipality with a [Municipal Revitalization Index](https://nj.gov/dca/home/MuniRevitIndex.html) (MRI) distress score of at least 50, can receive tax credits up to $60M
+:::callout{ header="Benefits" icon="false" calloutType="conditional" }
+
+- Total award cap of $42M, but residential projects also receiving Low-Income Housing Tax Credits (LIHTC), or projects located in certain economically disadvantaged locations may receive up to $60M
+- Tax credits equal to 45% of project costs up to $42M
+
+:::
+
+:::callout{ header="Bonuses" icon="false" calloutType="conditional" }
+
+- Commercial projects in a Government Restricted Municipalities (GRM) (These municipalities include: Atlantic City, Paterson and Trenton) can receive tax credits up to 50% of costs in project support
+- Newly constructed residential projects that are also utilizing 4% LIHTCs can receive tax credits for up to 60% of project costs
+- Projects using LIHTC or any other project in a `Qualified Incentive Track|qit-njeda` (QIT), GRM, or municipality with a [Municipal Revitalization Index](https://nj.gov/dca/home/MuniRevitIndex.html) (MRI) distress score of at least 50, can receive tax credits up to $60M
+
+:::
```

Comment:
Also might be a case where we wanted 1 large callout as opposed to 2 smaller ones potentially

Reply:
I'm going to leave this up to content to decide. If the pattern was two back-to-back green boxes, I want to preserve that.